src/Post.php

```diff
@@ -81,7 +82,7 @@ public function set_data_layer(): void
         } elseif ($is_new_ia && $this->is_get_informed_page()) {
             $this->data_layer['page_category'] = 'Get Informed Page';
         } elseif ($is_new_ia && $this->is_take_action_page()) {
-            $this->data_layer['page_category'] = 'Take Action Page';
```

Comment:
There might be a conflict here with [7045](https://jira.greenpeace.org/browse/PLANET-7045), it was changed in the corresponding [PR](https://github.com/greenpeace/planet4-master-theme/commit/e3813e12d607abe6bce66df401073efcf32e6e78#diff-5568aa3d3d5790f0967a994f11836016ec5d67d74c37cc62dc84ea56b39a8181R84) - but I guess Molly will decide in UAT

Reply:
Hey @lithrel today morning Molly tested it and passed the UAT, so I assume it is fine for her to use `Actions` instead. Also, we're going to modify the Get Informed page too.